### ISSUE TITLE
Joyboy should be responsive in all devices(mobile)

### ISSUE THREAD

**Issue by od-hunter** [keep-starknet-strange/joyboy#260]
• I feel it should be responsive to devices especially for mobiles
• It should fit in to the exact size frame of each mobile device
• For example, when I want to comment under another person’s post, have to swipe up before I’ll be able to see the comment box.
• For example, I don’t have to drag my screen to the left to see the like button clearly.

(The user profile page is responsive)

There are other instances but I’ll leave a few screenshots to better explain what I mean pictographically:

![IMG_6470](https://github.com/user-attachments/assets/c69e533d-93ac-447f-bd85-c40b22a49c35)
![IMG_6472](https://github.com/user-attachments/assets/ae3ea3e8-1dce-4605-8c18-09fef4f390c2)
![IMG_6473](https://github.com/user-attachments/assets/c7d6a7d5-fa7a-4d16-b418-a491f35d9d3d)
![IMG_6474](https://github.com/user-attachments/assets/3817d0cb-657d-4fb6-8c1c-0ac407693f14)

**Comment by ugur-eren** [comment]
This is not true, and I couldn't re-create the issue. I think you're accidentally zoomed-in or something.

**Comment by od-hunter** [comment]
I didn’t accidentally zoom it or out, but since you said it’s not true, my apologies.

**Comment by ugur-eren** [comment]
Oh, sorry.
What device are you using? Are you using any screen accessibility options [on the device](https://support.apple.com/guide/iphone/customize-the-text-size-and-zoom-setting-iphd6804774e/ios) or [on the safari](https://support.apple.com/guide/iphone/customize-your-safari-settings-iphb3100d149/ios#iphcc3ce3856) ?

**Comment by od-hunter** [comment]
Hello @ugur-eren , Thank you for noticing this bug. I’m using Safari and there’s no screen accessibility option enabled on the device.

**Comment by ugur-eren** [comment]
What device are you testing on?

**Comment by od-hunter** [comment]
An iPhone xr

**Comment by ugur-eren** [comment]
From your latest screenshot i can see that you've solved the responsive issue. Is that correct?

**Comment by od-hunter** [comment]
> From your latest screenshot i can see that you've solved the responsive issue. Is that correct?

No sir , the issue isn’t fixed yet. Like, the screen or displayed features isn’t supposed to be moving/be everywhere when one didn’t zoom in. I don’t have to drag my screen up to see the comment box. This is the best issue I can give. But if you don’t think this as an issue, let it be please. 

https://github.com/user-attachments/assets/1b88123a-15c8-4bd5-a5f4-da55d686d33a
